### ISSUE TITLE
insulation score

### ISSUE THREAD

**Issue by wawpaopao** [tanjimin/C.Origami#39]
Hello,

I noticed there was code for evaluation in issue 29, but I want to ask the specific code used for calculating the insulation score, ''import insulation as insu''.

Additionally, I am intrigued by the visualization presented in Figure 2.d of the ATAC_seq and CTCF. Are the values within each window_size region averaged to generate these plots?

Thanks!

**Comment by TeaQAQTea** [comment]
Hello jimin
I noticed that you have uploaded the code for the insulation score. However, I have a small question. The values in the matrix should mostly be positive, so the average values of the left and right regions should also be positive. In that case, the calculated insulation score should also be positive. Why is the insulation score in Fig. 2e of the article negative? Was it normalized in some way?

**Comment by tanjimin** [comment]
> Hello jimin
> I noticed that you have uploaded the code for the insulation score. However, I have a small question. The values in the matrix should mostly be positive, so the average values of the left and right regions should also be positive. In that case, the calculated insulation score should also be positive. Why is the insulation score in Fig. 2e of the article negative? Was it normalized in some way?

Yes, the figure is showing standardized insulation score for each window.

**Comment by TeaQAQTea** [comment]
> the figure is showing standardized insulation score for each window.

I would like to inquire whether the Pearson correlation coefficient (Pearson's r) on the graph represents the correlation coefficient between standardized data or between unstandardized data.

**Comment by tanjimin** [comment]
Hi @1944498970 I don't think standardization changes pearson/spearman correlations. I think the Pearson'r shown is calculated on unnormalized data.

**Comment by TeaQAQTea** [comment]
> Hi @1944498970 I don't think standardization changes pearson/spearman correlations. I think the Pearson'r shown is calculated on unnormalized data.

I am genuinely appreciative that your prompt and effective response has truly resolved my issue.
